lib/another_location_plugin.dart

```diff
@@ -5,9 +5,40 @@ import 'package:flutter/services.dart';
 class AnotherLocationPlugin {
   static const MethodChannel _channel =
       const MethodChannel('another_location_plugin');
+  static const EventChannel _permissionEventChannel =
+      EventChannel('permission_event_channel');
+  static Stream<dynamic> permissionStream =
+      _permissionEventChannel.receiveBroadcastStream();
 
   static Future<String?> get platformVersion async {
     final String? version = await _channel.invokeMethod('getPlatformVersion');
     return version;
   }
+
+  static Future<bool> get initializePlugin async {
+    final bool isInitialized = await _channel.invokeMethod('initializePlugin');
```

Comment:
You can add this String and the others than appears here into a String file

Reply:
Added to utils/Strings file